### PR TITLE
16.0 mig sale order warn message

### PR DESCRIPTION
**Pull request by aiendry-aktivsoftware**
### Standard migration to version 16.0

### CODE REVIEW

File: sale_order_warn_message/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module add a warning popup on sale to ensure warning is populated
```
Comment (rousseldenis) [inline sale_order_warn_message/readme/DESCRIPTION.rst]:
```suggestion
This module adds a warning popup on sale to ensure warning is populated
```

File: sale_order_warn_message/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+no only when partner is changed.
```
Comment (rousseldenis) [inline sale_order_warn_message/readme/DESCRIPTION.rst]:
```suggestion
not only when partner is changed.
```

File: sale_order_warn_message/views/sale_order.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <header position="after">
+                <div
+                    class="alert alert-warning"
+                    role="alert"
+                    attrs="{'invisible': [('sale_warn_msg', '=', False)]}"
+                    style="margin-bottom:0px;"
+                >
+                    <p>
+                        <i class="fa fa-info-circle" />
+                        &amp;nbsp;
```
Comment (rousseldenis) [inline sale_order_warn_message/views/sale_order.xml]: Missing a space here

image

File: sale_order_warn_message/readme/DESCRIPTION.rst (hunk 2 above)
Comment (rousseldenis) [reply sale_order_warn_message/readme/DESCRIPTION.rst]: The description should be enhanced as it is not a popup but a permanent alert warning on sale order form.

An image of the form should be great too (optional).

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@aiendry-aktivsoftware Don't forget that a meaningful PR title will help reviewers. Thanks

**Comment by dhara-wibtec** [comment]
@rousseldenis , I've made the changes you requested for please review for the same.
Thank You!

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by HaraldPanten** [comment]
Hi @aiendry-aktivsoftware could you rebase, please? THX

**Comment by rousseldenis** [comment]
@HaraldPanten Done

**Comment by HaraldPanten** [comment]
Closing as it has been superseded

**Pull request closed without merging**